contributing.md

```diff
@@ -128,34 +128,34 @@ classes based on the new schema.
 We take advantage of two tools to run tests:
 
 * [`tox`](https://tox.readthedocs.io/en/latest/), which is both a virtualenv management and test tool.
-* [`nose`](https://nose.readthedocs.org/en/latest/), which is is an extension of Python's unittest
+* [`pytest`](https://docs.pytest.org/en/latest/), a powerful framework for unit testing.
 
-### Running Tests with `nose`
+### Running Tests with `pytest`
 
 Since our tests cover *all* the functionality, to prevent tons of errors from showing up and having to parse through a messy output, you'll need to install `optional-requirements.txt` as explained above.
 
-After you've done that, go ahead and follow (y)our Nose!
+After you've done that, go ahead and run the test suite!
 
 ```bash
-nosetests -w  packages/python/plotly/plotly/tests/
+pytest  packages/python/plotly/plotly/tests/
 ```
 
 Or for more *verbose* output:
 
 ```bash
-nosetests -w  packages/python/plotly/plotly/tests/ -v
+pytest -v  packages/python/plotly/plotly/tests/
 ```
 
 Either of those will run *every* test we've written for the Python API. You can get more granular by running something like:
 
 ```bash
-nosetests -w  packages/python/plotly/plotly/tests/test_core/
+pytest  packages/python/plotly/plotly/tests/test_core/
 ```
 
 ... or even more granular by running something like:
 
 ```bash
-nosetests plotly/tests/test_plotly/test_plot.py
+pytest plotly/tests/test_plotly/test_plot.py
 ```
```

Comment:
Oh can we add here the incantation for running a single test? I think it’s with a colon? I always forget it!